experiments/simple_train_config.py

```diff
@@ -68,6 +68,12 @@ class SimpleTrainConfig:
     optimizer_config: OptimizerConfig | None = None
     """Optimizer configuration to use. If not set, Adam will be used."""
 
+    reinit_tokens: list[str] | bool = False
```

Comment:
Definitely not SimpleTrainConfig anymore...

Reply:
![image](https://github.com/user-attachments/assets/1c15d8d1-5b61-4030-948a-478cda3d543f)

Reply:
Returned it to as simple as it was before 😆